docs/guide/querying-data.md

```diff
@@ -15,4 +15,12 @@ export default Ember.Route.extend({
 
 We can combine `orderBy` with `limitToFirst`, `startAt`, `endAt`, and `equalTo` to construct more complex queries. See the documentation on Firebase's [querying methods](https://firebase.google.com/docs/database/web/retrieve-data#sorting_and_filtering_data) for more details.
 
+## Working with security rules
+
+In the above example, if the Real-time database implemented security rules which restricted access to some posts then the query would fail.
+
+We suggest changing your "schema" in cases were you want to restrict access and `findAll`. You can change the path the data is stored in the database by overriding `pathForType`.
```

Comment:
do you mean access **to** `findAll`?

suggest "you can change the path where data is stored by overriding `pathForType` in the adapter"